CHANGES.rst

```diff
@@ -1,34 +1,32 @@
 Changes
 =======
 
-dev (master)
```

Comment:
Is the dev section going away in master? Or was this meant to be merged into release?

Reply:
Per #1392 we won't be tagging releases on the release branch anymore, will be doing so on the master branch. Whenever we do additional development on master we'll have to add the `dev (master)` section back to whichever changelog entry comes next.

Reply:
Ah, didn't realize we'd be getting rid of `__version__ = 'dev'` in master. Historically, it has been very useful to know if the install version is directly from the repo vs a tagged release. Will the immediate next release also bump the `__version__` to an unreleased version too?

Don't think I mentioned that in #1392 but I won't stand in the way. :)